### DISCUSSION TITLE
Other price databases

### DISCUSSION THREAD

**Discussion by raphodn** [openfoodfacts/open-prices#450] [General]
A list of other price databases

|Name|Link|Description|Additional info|
|---|---|---|---|
|Tous Anti Inflation|https://play.google.com/store/apps/details?id=com.simeonov.tousantiinflation|French mobile app to compare food prices||
|Qui est le moins cher ?|https://www.quiestlemoinscher.leclerc/|E.Leclerc price comparison||
|Heisse Preise|https://heisse-preise.io/|Austrian-German food tracker||
|Reprice|https://reprice-app.fr/|French mobile app to compare food prices||
|Idealo|https://www.idealo.fr/|Online price tracker||
|Keepa|https://keepa.com/|Amazon price tracker||
|Kiprix|https://kiprix.com/|Comparateur Martinique vs Métropole||
|pingprice|https://www.pingprice.be/en/|Belgium comparator||
|lebondrive|https://lebondrive.fr/|Compare prices from french drives online shops||
|circl|https://www.circl.app/|French app working with extraction||
|PriceCatcher|https://pricecatcher.kpdn.gov.my/|Malaisian food tracker app seemingly promoted by the government||
|Prixing|(closed)|[Youtube video](https://youtu.be/MrvuE0TFjU8)||
|||||

### Related discussions

- #311

**Comment by serpico** [comment]
PriceCatcher https://pricecatcher.kpdn.gov.my/ Malaisian food tracker app seemingly promoted by the government ( Ministry of Domestic Trade and Costs of Living - KPDN )

**Reply by raphodn** [reply]
ajouté, merci !

**Reply by CloCkWeRX** [reply]
Sadly, blocked from some locations? I get connection refushed.

**Comment by raphodn** [comment]
@julia-gx found https://kassal.app, in Norway
@odin-h do you know about it ?

**Reply by odin-h** [reply]
Yes, I use it regularly to search for barcodes when contributing to Open Prices.
There's also:
* https://norges.online/
* https://enhver.no/
* https://matpris.ai/
* https://allematpriser.no/
